docs/roadmap.md

```diff
@@ -1,117 +1,56 @@
 # Roadmap
 
-The item listed here are proposed items for Argo Rollouts and are subject to change. To see where items
-may fall into releases, visit the [github milestones](https://github.com/argoproj/argo-rollouts/milestones)
-and notice if the item appears in the milestone description.
+The Argo Rollouts roadmap is maintained in Github Milestones on the Github repository.
 
-- [Roadmap](#roadmap)
-    - [Weight Verification](#weight-verification)
-    - [Istio Canary using DestinationRule subsets](#istio-canary-using-destinationrule-subsets)
-    - [Webhook Notifications](#webhook-notifications)
-    - [Rollback Window](#rollback-window)
-    - [Header Based Routing](#header-based-routing)
-    - [Shadow Traffic](#shadow-traffic)
+## Release Cycle
 
-# v1.0
+### Schedule
 
-## Weight Verification
+These are the upcoming releases date estimates:
 
-[Issue #701](https://github.com/argoproj/argo-rollouts/issues/701)
+| Release | Release Planning Meeting | Release Candidate 1   | General Availability     |
+|---------|--------------------------|-----------------------|--------------------------|
+| v1.4    | TBD                      | Monday, Dec. 19, 2022 | Tuesday, Jan. 9, 2023    |
+| v1.5    | Monday, Mar. 6, 2023     | Monday, Mar. 20, 2023 | Monday, Apr. 10, 2023    |
+| v1.6    | Monday, Jun. 5, 2023     | Monday, Jun. 19, 2023 | Wednesday, Jul. 12, 2023 |
+| v1.7    | Monday, Sep. 4, 2023     | Monday, Sep. 18, 2023 | Monday, Oct. 9, 2023     |
 
-When Argo Rollouts adjusts a canary weight, it currently assumes that the adjustment was made and
-moves on to the next step. However, for some traffic routing providers, this change can take a long
-time to take effect (or possibly never even made) since external factors may cause the change to
-become delayed.
+### Release Process
 
-This proposal is to add verification to the traffic routers so that after a setWeight step, the
-rollout controller could verify that the weight took effect before moving on to the next step. This
-is especially important for the ALB ingress controller which are affected by things like rate
-limiting, the ALB ingress controller not running, etc...
+#### Minor Releases (e.g. 1.x.0)
 
+A minor Argo Rollouts release occurs four times a year, once every three months. Each General Availability (GA) release is
+preceded by several Release Candidates (RCs). The first RC is released three weeks before the scheduled GA date. This
+effectively means that there is a three-week feature freeze.
```

Comment:
"feature freeze" to me means something else. General PRs can still be merged in master even after release branch cut. I would skip using this term to avoid confusion.

Reply:
Removed mention of it, was copied from argocd fyi